lib/Unicode/Properties/Core.hs

```diff
@@ -31,3 +33,9 @@ isUpper = P.isUppercase
 {-# INLINE isLower #-}
 isLower :: Char -> Bool
 isLower = P.isLowercase
+
+-- | Selects White_Space Unicode characters defined in Unicode character
+-- database.  This should be equivalent to "Data.Char.isSpace".
```

Comment:
"Select Unicode whitespace characters".
"Same as @Data.Char.isSpace@ from the base package".

Reply:
You might have something to say about the rest of the documentation. Can you check the docs in non-internal modules and change them for the better?

Reply:
If this is the only change then I'll change and push it.